.github/ISSUE_TEMPLATE/security-vulnerabilities-report.yml

```diff
@@ -1,34 +1,81 @@
+      Example: libX11 in `x11-lib` (7.7.20220817-1)
+    placeholder: '`package_name` version'
   validations:
     required: true
-- type: textarea
+- type: input
   attributes:
-    label: 'Other security advisory IDs'
-    description: 'Other security advisory IDs. Please list with references to each ID.'
+    label: 'CVE IDs'
+    description: 'Fill N/A if no CVE ID is assigned.'
+    placeholder: 'CVE-20xx-xxxx[, CVE-20xx-xxxx, ...]'
   validations:
     required: true
+# Severity is a new field of the template. This may help maintainers to identify which issue is most urgent and require attentions.
+# Note that a `None` indicates that the severity is under evaluation now, and not recommended.
+- type: dropdown
+  attributes:
+    label: 'Severity'
+    description: | 
+      Select the severity of reported vulnerability(ies). Refer to our wiki (not ready yet) for details.
+      Choose the highest severity out of reported vulnerabilities.
+    options:
+      - 'Critical/0-day'
+      - 'High'
+      - 'Moderate'
+      - 'Low'
 - type: textarea
   attributes:
-    label: 'Description'
-    description: 'Original mailing list message(s) or other description here.'
+    label: 'Other security advisory IDs'
+    description: 'Other security advisory(ies), mailing list message(s) or bug report(s). Please list with reference(s) to each ID.'
+    placeholder: |
+      Examples:
+      - Debian: [DSA-xxxx-x](link_to_DSA)
+      - Mail list: (link_to_mail_list_archive)
+      - Bug report: (link_to_upstream_bug_report)
   validations:
     required: true
 - type: textarea
   attributes:
-    label: 'Patches'
-    description: 'Enter "N/A" if version update is preferred, otherwise please label each patch with ref. to original vulnerability ID(s).'
+    label: 'Description'
+    description: |
+      Original mailing list message(s) or other description may be listed here. 
+      You may also include supplemental details:
+      - Version where the bug was introduced
+      - Possible security impact on AOSC OS
+      - Available PoCs (required if the reporting target is an AOSC-developed component)
+      - Reasons to flag selected severity
+      - CWE (Common Weakness Enumeration), if applicable
+      - ...
   validations:
     required: true
+# The original `patches` is renamed to `Fixed version or patches`. If an package update is suggested the reporter should include the latest version and the earliest fixed version if applicable. 
 - type: textarea
   attributes:
-    label: 'PoC(s)'
-    description: 'Please list links to available PoCs (Proofs of Concept).'
+    label: 'Patch(es)/Solution(s)'
+    description: | 
+      Enter N/A to indicate that a regular upgrade to latest version is favoured. Otherwise, fill this field with one or more of the following:
+      - Patch(es), if a backport is advised or no new upstream release is available
+      - Specific version, if upgrading to latest requires extra attention (please specify)
```

Comment:
A specific non-latest version, if upgrading to the latest version may cause issue (please specify)